services/src/unit-test/java/uk/gov/moj/sdt/services/BulkFeedbackServiceTest.java

```diff
@@ -248,4 +221,22 @@ private void addValidIndividualRequest(final IBulkSubmission bulkSubmission, fin
         bulkSubmission.addIndividualRequest(individualRequest);
     }
 
+    @Test
+    void testSetBulkSubmissionDao(){
+        BulkFeedbackService bulkFeedbackServiceMock = mock(BulkFeedbackService.class);
+
+        bulkFeedbackServiceMock.setBulkSubmissionDao(mockBulkSubmissionDao);
+
+        verify(bulkFeedbackServiceMock).setBulkSubmissionDao(mockBulkSubmissionDao);
+    }
+
+    @Test
+    void testSetGlobalParametersCache(){
+        BulkFeedbackService bulkFeedbackServiceMock = mock(BulkFeedbackService.class);
+
+        bulkFeedbackServiceMock.setGlobalParametersCache(mockGlobalParameterCache);
+
+        verify(bulkFeedbackServiceMock).setGlobalParametersCache(mockGlobalParameterCache);
+    }
```

Comment:
These tests aren't testing the class, they're testing the mocked version of the class.

To test that a private field has been set first use the corresponding set method to set it, then use the getAccessibleField() method on the AbstractSdtUnitTestBase class to get it.  You can then use an assertEquals() to check that it's been set to the expected object.

Reply:
Done